### DISCUSSION TITLE
Welcome to x-bot-sweeper Discussions!

### DISCUSSION THREAD

**Discussion by sleeyax** [sleeyax/x-bot-sweeper#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Liam101x** [comment]
Any progress on enabling the scanning of large accounts e.g. 200k followers?

**Reply by sleeyax** [reply]
I currently lack the time to work on this project myself unfortunately. I was hoping to see more contributions from the community.

**Comment by Liam101x** [comment]
Thanks for the work you've done so far. With the infestation of x by bots I
think it has great commercial potential.
